contract/src/offer-up.contract.js

```diff
@@ -129,19 +134,10 @@ export const start = async zcf => {
     return 'trade complete';
   };
 
-  /**
-   * Make an invitation to trade for items.
-   *
-   * Proposal Keywords used in offers using these invitations:
-   *   - give: `Price`
-   *   - want: `Items`
-   */
-  const makeTradeInvitation = () =>
-    zcf.makeInvitation(tradeHandler, 'buy items', undefined, proposalShape);
-
-  // Mark the publicFacet Far, i.e. reachable from outside the contract
-  const publicFacet = Far('Items Public Facet', {
-    makeTradeInvitation,
+  // Use zone.exo to make a publicFacet suitable for use by remote callers.
+  const publicFacet = zone.exo('Items Public Facet', undefined, {
+    makeTradeInvitation: () =>
+      zcf.makeInvitation(tradeHandler, 'buy items', undefined, proposalShape),
```

Comment:
by "concise method syntax" we mean what MDN calls [Method definitions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions):

```suggestion
    makeTradeInvitation() {
      return zcf.makeInvitation(tradeHandler, 'buy items', undefined, proposalShape),
    }
```

Reply:
Yes. Arrow functions are not `this` sensitive. More precisely, they get their `this` binding lexically rather than from their caller. Exo methods in general need to be `this` sensitive, and so should use concise method syntax.

Reply:
Oh, I totally missed that point. Let me fix that.